### ISSUE TITLE
All validation loss reported as 0.0000e+00 in keras / TF

### ISSUE THREAD

**Issue by becker929** [googlecolab/colabtools#1472]
Bug report for Colab: http://colab.research.google.com/.

For questions about colab usage, please use [stackoverflow](https://stackoverflow.com/questions/tagged/google-colaboratory/).

- Describe the current behavior:
During training, val_loss is reported as 0.0000e+00. 
When printing model.history.history['val_loss'], an array of [0.0,..., 0.0] is printed.
However, when evaluating the model on examples val_x, val_y, a non-zero loss is reported.
This happens for at least 4 different notebooks, including notebooks that worked properly earlier today. 

- Describe the expected behavior:
During training, a non-zero validation loss should be reported in most cases. 
These non-zero losses should be reported in the model history. 
When evaluating the model against the validation examples val_x, val_y, a loss similar to that reported in the model history should be reported. 

- The web browser you are using (Chrome, Firefox, Safari, etc.):
Chrome 84.0.4147.105 

- Link (not screenshot!) to a *minimal, public, self-contained* notebook that
  reproduces this issue (click the *Share* button, then *Get Shareable Link*):
https://colab.research.google.com/drive/1YP_dR7O9KsqSeLyrgFTzOSna0EggL9Gs?usp=sharing

**Comment by craigcitro** [comment]
Thanks for the excellent bug report -- could I ask you to report this upstream on the TF repo?

**Comment by becker929** [comment]
Hi Craig, I did and it’s being looked into.

**Comment by craigcitro** [comment]
Awesome -- closing this as a duplicate of https://github.com/tensorflow/tensorflow/issues/42053.
